exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_client.h

```diff
@@ -13,34 +13,52 @@
 
 #include "opentelemetry/exporters/otlp/otlp_grpc_client_options.h"
 
+// clang-format off
```

Comment:
When we have no blank lines between prefix/suffix guard headers and generated .pb.h headers. clang-format will reorder the include sentences. We can add this to avoid these mistakes.